### PR TITLE
DOC-2890 OU field is optional part of x500 name, OS and CE 4.6 - 4.8 fix

### PR DESCRIPTION
**Pull request by erinking-r3**

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.6/node/setup/node-naming.md [hunk 1]
```diff
@@ -39,8 +39,8 @@ present in the majority of names, but is an option for the cases which require i
 The name must also obey the following constraints:
 
 
-* The `organisation`, `organisational-unit`, `locality`, and `country` attributes are present
-* The `state` and `common name` attributes are optional
+* The `organisation`, `locality`, and `country` attributes are present
+* The `state` `organisational unit` and `common name` attributes are optional
```
Comment (benpesterr3) [inline content/en/docs/corda-enterprise/4.6/node/setup/node-naming.md:43]: this is a dumb question maybe - but these are in code snippets, but have spaces in them. Are the attributes in code written with a space or in `camelCase` ?

Comment (erinking-r3) [reply content/en/docs/corda-enterprise/4.6/node/setup/node-naming.md:43]: missed the -
```suggestion
* The `state` `organisational-unit` and `common name` attributes are optional
```

File: content/en/docs/corda-enterprise/4.7/node/setup/node-naming.md [hunk 2]
```diff
@@ -39,8 +39,8 @@ present in the majority of names, but is an option for the cases which require i
 The name must also obey the following constraints:
 
 
-* The `organisation`, `organisational-unit`, `locality`, and `country` attributes are present
-* The `state` and `common name` attributes are optional
+* The `organisation`, `locality`, and `country` attributes are present
+* The `state` `organisational unit` and `common name` attributes are optional
```
Comment (benpesterr3) [inline content/en/docs/corda-enterprise/4.7/node/setup/node-naming.md:43]:
```suggestion
* The `state` `organisational-unit` and `common name` attributes are optional
```

**Review by benpesterr3 — COMMENTED** [review]
Some more still missing I think - also is it the same thing for common name? should it be hyphened?

File: content/en/docs/corda-enterprise/4.6/node/setup/node-naming.md [hunk 3]
```diff
@@ -39,8 +39,8 @@ present in the majority of names, but is an option for the cases which require i
 The name must also obey the following constraints:
 
 
-* The `organisation`, `organisational-unit`, `locality`, and `country` attributes are present
-* The `state` and `common name` attributes are optional
+* The `organisation`, `locality`, and `country` attributes are present
+* The `state` `organisational-unit` and `common name` attributes are optional
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.6/node/setup/node-naming.md:43]:
```suggestion
* The `state`, `organisational-unit`, and `common name` attributes are optional.
```

File: content/en/docs/corda-enterprise/4.7/node/setup/node-naming.md [hunk 4]
```diff
@@ -39,8 +39,8 @@ present in the majority of names, but is an option for the cases which require i
 The name must also obey the following constraints:
 
 
-* The `organisation`, `organisational-unit`, `locality`, and `country` attributes are present
-* The `state` and `common name` attributes are optional
+* The `organisation`, `locality`, and `country` attributes are present
+* The `state` `organisational-unit` and `common name` attributes are optional
```
Comment (erinking-r3) [inline content/en/docs/corda-enterprise/4.7/node/setup/node-naming.md:43]:
```suggestion
* The `state` `organisational-unit` and `common name` attributes are optional.
```

File: content/en/docs/corda-enterprise/4.6/node/setup/node-naming.md [hunk 5]
```diff
@@ -39,8 +39,8 @@ present in the majority of names, but is an option for the cases which require i
 The name must also obey the following constraints:
 
 
-* The `organisation`, `organisational-unit`, `locality`, and `country` attributes are present
-* The `state` and `common name` attributes are optional
+* The `organisation`, `locality`, and `country` attributes are present
+* The `state`, `organisational-unit`, and `common name` attributes are optional.
 * The maximum number of characters in the whole x500 name string is 128 characters
```
Comment (erinking-r3) [inline content/en/docs/corda-enterprise/4.6/node/setup/node-naming.md:44]:
```suggestion
* The `organisation`, `locality`, and `country` attributes are present.
* The `state`, `organisational-unit`, and `common name` attributes are optional.
* The maximum number of characters in the whole x500 name string is 128 characters.
```

File: content/en/docs/corda-enterprise/4.8/node/setup/node-naming.md [hunk 6]
```diff
@@ -39,8 +39,8 @@ present in the majority of names, but is an option for the cases which require i
 The name must also obey the following constraints:
 
 
-* The `organisation`, `organisational-unit`, `locality`, and `country` attributes are present
-* The `state` and `common name` attributes are optional
+* The `organisation`, `locality`, and `country` attributes are present
+* The `state` `organisational unit` and `common name` attributes are optional
```
Comment (erinking-r3) [inline content/en/docs/corda-enterprise/4.8/node/setup/node-naming.md:43]:
```suggestion
* The `organisation`, `locality`, and `country` attributes are present.
* The `state` `organisational unit` and `common name` attributes are optional.
```

File: content/en/docs/corda-os/4.6/node-naming.md [hunk 7]
```diff
@@ -44,8 +44,8 @@ present in the majority of names, but is an option for the cases which require i
 The name must also obey the following constraints:
 
 
-* The `organisation`, `organisational-unit`, `locality`, and `country` attributes are present
-* The `state` and `common name` attributes are optional
+* The `organisation`, `locality`, and `country` attributes are present
+* The `state` `organisational unit` and `common name` attributes are optional
 * The maximum number of characters in the whole x500 name string is 128 characters
```
Comment (erinking-r3) [inline content/en/docs/corda-os/4.6/node-naming.md:49]:
```suggestion
* The `organisation`, `locality`, and `country` attributes are present.
* The `state` `organisational unit` and `common name` attributes are optional.
* The maximum number of characters in the whole x500 name string is 128 characters.
```

File: content/en/docs/corda-os/4.7/node-naming.md [hunk 8]
```diff
@@ -44,8 +44,8 @@ present in the majority of names, but is an option for the cases which require i
 The name must also obey the following constraints:
 
 
-* The `organisation`, `organisational-unit`, `locality`, and `country` attributes are present
-* The `state` and `common name` attributes are optional
+* The `organisation`, `locality`, and `country` attributes are present
+* The `state` `organisational unit` and `common name` attributes are optional
 * The maximum number of characters in the whole x500 name string is 128 characters
```
Comment (erinking-r3) [inline content/en/docs/corda-os/4.7/node-naming.md:49]:
```suggestion
* The `organisation`, `locality`, and `country` attributes are present.
* The `state` `organisational unit` and `common name` attributes are optional.
* The maximum number of characters in the whole x500 name string is 128 characters.
```

File: content/en/docs/corda-os/4.8/node-naming.md [hunk 9]
```diff
@@ -44,8 +44,8 @@ present in the majority of names, but is an option for the cases which require i
 The name must also obey the following constraints:
 
 
-** The `organisation`, `organisational-unit`, `locality`, and `country` attributes are present
-* The `state` and `common name` attributes are optional
+* The `organisation`, `locality`, and `country` attributes are present
+* The `state` `organisational unit` and `common name` attributes are optional
 * The maximum number of characters in the whole x500 name string is 128 characters
```
Comment (erinking-r3) [inline content/en/docs/corda-os/4.8/node-naming.md:49]:
```suggestion
* The `organisation`, `locality`, and `country` attributes are present.
* The `state` `organisational unit` and `common name` attributes are optional.
* The maximum number of characters in the whole x500 name string is 128 characters.
```

**Review by erinking-r3 — COMMENTED** [review]
adding full stops to list items

File: content/en/docs/corda-enterprise/4.7/node/setup/node-naming.md [hunk 10]
```diff
@@ -39,8 +39,8 @@ present in the majority of names, but is an option for the cases which require i
...
 * The maximum number of characters in the whole x500 name string is 128 characters
```
[the remark this answers is not in the document]
Comment (ivanterziev-r3) [reply content/en/docs/corda-enterprise/4.7/node/setup/node-naming.md:44]: still missing commas

File: content/en/docs/corda-enterprise/4.6/node/setup/node-naming.md [hunk 11]
```diff
@@ -39,8 +39,8 @@ present in the majority of names, but is an option for the cases which require i
 The name must also obey the following constraints:
 
 
-* The `organisation`, `organisational-unit`, `locality`, and `country` attributes are present
-* The `state` and `common name` attributes are optional
+* The `organisation`, `locality`, and `country` attributes are present.
+* The `state`, `organisational-unit`, and `common name` attributes are optional.
 * The maximum number of characters in the whole x500 name string is 128 characters
```
Comment (erinking-r3) [inline content/en/docs/corda-enterprise/4.6/node/setup/node-naming.md:44]:
```suggestion
* The maximum number of characters in the whole x500 name string is 128 characters.
```

File: content/en/docs/corda-enterprise/4.8/node/setup/node-naming.md [hunk 12]
```diff
@@ -39,8 +39,8 @@ present in the majority of names, but is an option for the cases which require i
 The name must also obey the following constraints:
 
 
-* The `organisation`, `organisational-unit`, `locality`, and `country` attributes are present
-* The `state` and `common name` attributes are optional
+* The `organisation`, `locality`, and `country` attributes are present.
+* The `state`, `organisational unit`, and `common name` attributes are optional.
 * The maximum number of characters in the whole x500 name string is 128 characters
```
Comment (erinking-r3) [inline content/en/docs/corda-enterprise/4.8/node/setup/node-naming.md:44]:
```suggestion
* The maximum number of characters in the whole x500 name string is 128 characters.
```

Comment (erinking-r3) [inline content/en/docs/corda-enterprise/4.8/node/setup/node-naming.md]:
```suggestion
* The maximum number of characters in the whole x500 name string is 128 characters.
```

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

**Pull request merged**